main.js

```diff
@@ -0,0 +1,14 @@
+document.querySelector("header nav ul li").remove();
+document.querySelector(".featured span").remove();
+
+const h3 = document.querySelectorAll("main .posts article");
+for (let el of h3) {
+  if (el.textContent.includes("Stop Planning")) {
+    el.remove();
+  }
+}
```

Comment:
Interesting way to do this one! I like that it will always target the "Stop Planning" article no matter its position in the DOM. 
Here's another way to do this if removing the **_last element_** specifically is the main priority:
`h3[h3.length - 1].remove()`